### ISSUE TITLE
RSS feed from journal.tei-c.org missing

### ISSUE THREAD

**Issue by martindholmes** [TEIC/infrastructure#1]
If you go to the JTEI site on lodel:

https://journals.openedition.org/jtei/

you'll see this error message in the right-side panel:

~~~~~
Warning: Erreur de connection RSS sur l'url http://journal.tei-c.org/journal/gateway/plugin/AnnouncementFeedGatewayPlugin/rss2: cURL error 22: The requested URL returned error: 404 Not Found
~~~~~

This is presumably something that used to be provided by the old OJS but isn't set up on the new one.

**Comment by rvdb** [comment]
Hi @martindholmes ,

Could you try and see if that announcement section is still visible when logged out? Unless my browser's cache is misleading me, it seems that the feed is not shown on the public site. 

Apparently, the AnnouncementsFeed was present in OJS-2, but no longer in OJS-3. Although I find it in the [OJS source code](https://github.com/pkp/ojs/tree/master/plugins/generic/announcementFeed), no such folder is present in the latest code download. I want to inquire further, but if it can't be restored, we could probably remove these announcements to avoid further confusion. I've tried to "unpublish" it, but that does not seem to make it go away.

**Comment by martindholmes** [comment]
@rvdb You're right, I don't see it when I'm not logged in, so it's not urgent at all.

**Comment by peterstadler** [comment]
This issue is rather old so I take the liberty to close it. Please feel free to reopen though …
